drop-in/src/main/java/com/adyen/checkout/dropin/service/SessionDropInService.kt

```diff
@@ -79,6 +81,7 @@ open class SessionDropInService : DropInService(), SessionDropInServiceInterface
                     onSuccess = { response ->
                         val action = response.action
                         val result = when {
+                            response.isRefused() -> DropInServiceResult.Error(reason = response.resultCode)
```

Comment:
Does this terminate drop-in or not? I'm curious as to how we should handle this case

Reply:
No, it does not. With `DropInServiceResult.Finished` it would terminate, but I don't think that's nice UX. In this case, with `Error`, we show a dialog with a generic error message

Reply:
`Error` can also terminate drop-in but you should pass a `terminateDropIn = true` param to it. In all cases i agree that we shouldn't terminate it here.